apps/sms/js/thread_ui.js

```diff
@@ -2531,10 +2531,10 @@ var ThreadUI = {
       // eg. type "a", then delete it.
       // Always remove the the existing results.
       this.toggleRecipientSuggestions();
-      return;
+      return Promise.resolve(null);
```

Comment:
Well, I partly agree, I have mixed feelings, IMO here is ```exception``` case since we don't pass correct arguments to ```searchContacts```, but we have another problem here ```toggleRecipientSuggestions``` call in ```searchContacts``` - doesn't look like it's correct place for such call. You asking to search for something, but it can change UI :)